src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -400,12 +403,19 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
         {
             try
             {
+                // do not send invocation requests for functions that failed to load or for functions that could not be indexed by the worker
                 if (_functionLoadErrors.ContainsKey(context.FunctionMetadata.GetFunctionId()))
                 {
                     _workerChannelLogger.LogDebug($"Function {context.FunctionMetadata.Name} failed to load");
                     context.ResultSource.TrySetException(_functionLoadErrors[context.FunctionMetadata.GetFunctionId()]);
                     _executingInvocations.TryRemove(context.ExecutionContext.InvocationId.ToString(), out ScriptInvocationContext _);
                 }
+                else if (_metadataRequestErrors.ContainsKey(context.FunctionMetadata.GetFunctionId()))
+                {
+                    _workerChannelLogger.LogDebug($"Function {context.FunctionMetadata.Name} failed to load");
```

Comment:
Make this message slightly different from above -- like `... failed to load metadata from worker.`

Reply:
Latest commit should address this